roles/openshift_health_checker/library/docker_info.py

```diff
@@ -0,0 +1,50 @@
+# pylint: disable=missing-docstring
+"""
+Ansible module for determining information about the docker host.
+"""
+
+from ansible.module_utils.docker_common import *
```

Comment:
uggh... my comments apply to that file as well -- not a good source for learning Python ;)